packages/lab/src/dropdown-next/DropdownNext.css

```diff
@@ -1,108 +1,107 @@
-.saltDropdownNext-button {
-  --dropdownNext-borderColor: var(--salt-editable-borderColor);
-  --dropdownNext-borderStyle: var(--salt-editable-borderStyle);
-  --dropdownNext-borderWidth: var(--salt-size-border);
+.saltDropdownNext {
+  --dropdown-next-borderColor: var(--salt-editable-borderColor);
+  --dropdown-next-color: var(--salt-content-primary-foreground);
+  --dropdown-next-cursor: var(--salt-selectable-cursor-hover);
+}
 
-  align-items: center;
-  background: var(--saltDropdownNext-background, var(--dropdownNext-background));
-  color: var(--saltDropdownNext-color, var(--salt-content-primary-foreground));
+.saltDropdownNext-button {
+  all: unset;
```

Comment:
Not sure about this. Seems to be causing a change to Dropdown which makes it align differently to other form controls, but `all: unset` makes it difficult to tell exactly what's being overridden, and this can't be 'undone' by consumers.

Probably better to to override/unset specific css properties which you're finding problematic